universal-login-sdk/lib/config/SdkConfig.ts

```diff
@@ -4,4 +4,6 @@ export interface SdkConfig {
   paymentOptions: PaymentOptions;
   observedTokensAddresses: string[];
   observedCurrencies: ObservedCurrency[];
+  authorisationsObserverTick: number;
```

Comment:
`authorisationsObserver: {tick: number, timeout: number}`

Reply:
` authorisationsObserverTick: number` and `executionFactoryTick: number` should be optional